### PR TITLE
Show warning message for experimental API.

### PR DESCRIPTION
**Pull request by huningxin**

### CODE REVIEW

File: xwalk/common/common_api.js [hunk 1]
```diff
@@ -167,21 +167,23 @@ var Common = function() {
...
-  // _addMethod(name, has_callback):
+  // _addMethod(name, has_callback, experimental?):
   //     Convenience function for adding methods to an object that have a
   //     correspondent on the native side. Methods names that start with "_" are
   //     define as not enumerable by default. Set |has_callback| to true if the
   //     method expects a callback as the last parameter.
+  //     Set |experimental| to show warning message to console.
   //
-  // _addMethodWithPromise(name, promise, wrapArgs?, wrapReturns?):
+  // _addMethodWithPromise(name, promise, wrapArgs?, wrapReturns?, experimental?):
```
Comment (qjia7) [inline xwalk/common/common_api.js]: I forgot to change the description for PR #31. So please also add <code>wrapErrorResult?</code>
```
_addMethodWithPromise(name, promise, wrapArgs?, wrapReturns?, wrapErrorResult?, experimental?)
``` 
Same to below methods.

**Comment by qjia7** [comment]
LGTM with nit. I am not in office tomorrow. If you resolved the comment, please feel free to merge it.

**Comment by huningxin** [comment]
Fixed the comments.

**Comment by huningxin** [comment]
Merging.

**Pull request merged**